Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectDetails.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+import Kingfisher
+import KsApi
+import SwiftUI
+
+struct PPOProjectDetails: View {
+  let imageUrl: URL?
+  let title: String?
+  let pledge: GraphAPI.MoneyFragment
+
+  var body: some View {
+    HStack {
+      KFImage(self.imageUrl)
+        .resizable()
+        .clipShape(Constants.imageShape)
+        .aspectRatio(Constants.imageAspectRatio, contentMode: .fit)
+        .frame(width: Constants.imageWidth)
+      Spacer()
+        .frame(width: Constants.spacing)
+      VStack {
+        if let title {
+          Text(title)
+            .font(Font(Constants.titleFont))
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .lineLimit(2)
+        }
+        // TODO: Localize
+        Text("\(self.pledge.symbol ?? "")\(self.pledge.amount ?? "") pledged")
```

Comment:
nit: Should this also have an `if let` and only show if there is a pledge symbol and amount? " pledged" would look pretty weird, but I'm okay with it since it really shouldn't happen regardless.